R/environment.R

```diff
@@ -0,0 +1,34 @@
+# Install and load packages for R scripts -----------------------------------------------
+
+cran_pkgs <- c(
+
+  # config
+  "ConfigParser",
+
+  # General
+  "dplyr", "tidyr", "stringr", "readr", "here", "tibble", "fs",
+
+  # Testing
+  "assertr"
+)
+
+
+to_install <- cran_pkgs[!cran_pkgs %in% installed.packages()]
+
+if (length(to_install) > 0) {
+  install.packages(to_install, deps = TRUE, repos = "https://cran.r-project.org")
+}
+
+# non-CRAN packages
+if (!"unpaywallR" %in% installed.packages()) {
+  devtools::install_github("NicoRiedel/unpaywallR")
+}
+
+invisible(lapply(c(cran_pkgs), library, character.only = TRUE))
+
+# Specify configuration  -----------------------------------------------
```

Comment:
Hey @delwen ! Having trouble reproducing this part. I've installed ConfigParser but get an error that "config.ini" doesn't exist. I believe this is where you're storing credentials? In that case, maybe we should chat (offline) about the unpaywall credentials you're using.